src/wms/src/test/java/org/geoserver/wms/wms_1_3/LegendCapabilitiesTest.java

```diff
@@ -147,13 +147,13 @@ public void testCapabilities() throws Exception {
         // three
         // cases
         String expectedGetLegendGraphicRequestURL =
-                "/ows?service=WMS&request=GetLegendGraphic&format=image%2Fjpeg&width=20&height=20&layer=gs%3A"
+                "/ows?service=WMS&request=GetLegendGraphic&format=image%2Fjpeg&width=22&height=22&layer=gs%3A"
```

Comment:
There are constants for these values, best to use them

Reply:
fixed